### ISSUE TITLE
Looks bad in light mode

### ISSUE THREAD

**Issue by rdbende** [tukaan/tukaan.github.io#5]
The main page looks bad in light mode. Everything is the same shade of gray. Needs some white.

![image](https://user-images.githubusercontent.com/77941087/199696157-713f9e1c-33a9-4eaf-9ea3-1f2d3b59f515.png)

Syntax highlighting is also quite bad with light theme.

**Comment by rdbende** [comment]
Probably the dotted background should be inverted.

**Comment by Moosems** [comment]
There's a dark mode?

**Comment by rdbende** [comment]
Yeah. It automatically follows the system color scheme, without a button to change it manually.

**Comment by Moosems** [comment]
I tried getting it to change my changing my system theme and it didn't work :(
